core/embed/models/T2B1/model_T2B1.h

```diff
@@ -24,15 +24,15 @@
 
 #define IMAGE_CHUNK_SIZE (128 * 1024)
 #define IMAGE_HASH_BLAKE2S
-#define BOARD_CAPABILITIES_ADDR 0x0800BF00
-#define CODE_ALIGNMENT 0x200
+#define DISPLAY_JUMP_BEHAVIOR DISPLAY_RETAIN_CONTENT
```

Comment:
@TychoVrahe These `model_xxx.h` headers are starting to get a bit messy. I don't think we need to address it right now, but we could group related symbols together and unify the order of symbols across all these files. Maybe in some subsequent PR...